### PR TITLE
[Feature/#59] splash / marker / app icon / status, navigation bar

### LINKED ISSUE #59: [Feature] Splash Screen
## 🚩 TO DO
- [x] Splash 구현

### PR DESCRIPTION
**Pull request by chattymin**
- closed #59 

## *⛳️ Work Description*
- init splash screen
- add marker
- add app icon
- add status, navigation bar color

## *📸 Screenshot*

https://github.com/user-attachments/assets/cdd875a8-26a9-43bc-b7f9-8a3205285c03

## *📢 To Reviewers*
-  완료~

### CODE REVIEW

File: feature/main/src/main/java/com/hankki/feature/main/splash/SplashScreen.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.weight(205f))
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            Image(
+                imageVector = ImageVector.vectorResource(id = R.drawable.img_subtitle),
+                contentDescription = "subtitle"
+            )
+            Spacer(modifier = Modifier.height(14.dp))
+            Image(
```
Comment (0se0) [inline feature/main/src/main/java/com/hankki/feature/main/splash/SplashScreen.kt:34]: spacer 띄워주시면 좋을 거 같아욥

**Review by 0se0 — APPROVED** [review]
정말 빠르게 구현해주셨네요! 수고하셨습니다~

**Review by hyeeum — APPROVED** [review]
고생하셨습니당 :) !
코멘트 달게 딱히 없어요..............................................................................쩜쩜,,,,,

File: feature/main/src/main/java/com/hankki/feature/main/splash/SplashScreen.kt (hunk 1 above)
Comment (chattymin) [reply feature/main/src/main/java/com/hankki/feature/main/splash/SplashScreen.kt]: 넴~

**Pull request merged**